integration_tests/CMakeLists.txt

```diff
@@ -532,6 +532,7 @@ RUN(NAME test_str_02         LABELS cpython llvm llvm_jit c)
 RUN(NAME test_str_03         LABELS cpython llvm llvm_jit c)
 RUN(NAME test_str_04         LABELS cpython llvm llvm_jit c wasm)
 RUN(NAME test_str_05         LABELS cpython llvm llvm_jit c)
+RUN(NAME test_string_01      LABELS cpython llvm llvm_jit c)
```

Comment:
Either rename this to `test_string`, or (rather) rename the file `test_string.py` to `test_string_01.py`.